ontology/pso.md

```diff
@@ -15,7 +15,9 @@ contact:
   github: cooperl09
   orcid: 0000-0002-6379-8932
 description: The Plant Stress Ontology describes...
-domain: plant disease and abiotic stress
+domain: agriculture
+tags:
+  - plant disease and abiotic stress
```

Comment:
are these two different tags?

Reply:
Probably supposed to be a "conjunctive" expression, I think we can split this like the rest.

Reply:
Split it!

Reply:
I meant, I splitted it.